README.md

```diff
@@ -268,6 +268,11 @@ A variety of projects, a limited subset of which I know about below.
 
 * Aquamarine, a distributed choreography language & platform
 
+
+## [Fuel Labs](https://jobs.lever.co/fuellabs)
```

Comment:
You may want to add the 📤 icon to indicate these jobs are remote friendly.

Reply:
Thanks, updated.